test/specs/token-assert.js

```diff
@@ -66,10 +66,12 @@ describe('modules/token-assert', function() {
                 nextToken: tokens[1]
             });
 
-            assert(!onError.called);
+            assert(onError.notCalled);
         });
+    });
 
-        it('should trigger error on invalid space count between tokens', function() {
+    describe('spacesBetween', function() {
```

Comment:
It seems like spacesBetween and linesBetween should have extremely similar tests. Perhaps not in logic, but definitely in structure. For example. Verify error messages, verify exactly logic, verify atLeast logic, etc. I think the structure of the tests should be made to match. It looks like this is missing tests for missing other options that linesBetween has.